setup.py

```diff
@@ -13,7 +13,7 @@
 
 requirements = [
     'pylokit==0.8.1',
-    'django>=1.8,<1.12',
+    'django>=1.8,<=2.2.13',
```

Comment:
the change you made above to `MIDDLEWARE` was only introduced with Django 1.10.

For simplicity let's drop 'support' for Django 1.

Also 2.2.13 is too strict. We'd need to update this library after every DJ update. I recommend just set it to <3.0.0